### DISCUSSION TITLE
Umbral Reach? and a bit of feedback

### DISCUSSION THREAD

**Discussion by cryptoatropine** [endless-sky/endless-sky#9878] [Q&A]
Umbral Reach ?? .... so i am assuming that this is or is going to be related to the crazy voices planets ?? in the far east ?? 

I just wanted to know if there is anything more to it yet??  also as a note here i have sometime, not great with coding cpp, however i seem to understand the story texts format, mostly, and could potentially help build story..  if someone wants help with ideas i could probably toss a few out..    testing i can do, but i hate filling reports .. 

Also i have been watching this game since almost day one,   it has come along by leaps and bounds ...  good work and high standards i notice, and keep that up. 

the only thing i find sorta bothersome, is when quests end abruptly  I think Umbral Reach, is one such example. 

Side question: 
for quests started by spaceport button,   is there some sorta timer or something ? i mean sometimes fly around 'forever' pressing spaceport nothing, then lots all in a row ??   linked to combat or something ??

**Comment by Saugia** [comment]
Yes, the Umbral Reach is Rulei-related. I have a decently-sized outline regarding Rulei lore, which includes the Umbral Reach (the area of dark systems) and Acheron (the overall area the Gegno inhabit). While the Rulei's story is at an end, I plan on opening the history of it up bit by bit throughout the game for lore exposition. At the moment, there's only that little bit in-game, but I have plenty of planned content for it.

Specifically, the "Umbral Reach" discussion triggers if you have entered the area. The Rulei *Solemnity* will know you have, and drop that comment on you.

Basically, it's newer content that will be added upon as time goes on. This is normal in Endless Sky's development - all factions, save for human space of course, were once at a skeletal point in-game, but were added to over time. The same will be true for the Rulei, although a bit more eerie and discreetly. Rather than a solidified in-depth story, the tale of these ancients will be done arbitrarily, similar to that of the Drak.

You just have to stay tuned until then. :)

Regarding the spaceport question - it is simply random chance. Every landing is treated as that random chance - the time you spend in-space or traveling doesn't matter, but every time you land (anywhere) is a different roll, so to speak. Some missions, however, may only offer if you're a decent ways away from its source, but these are few and far between and typically part of the later game content. Another possibility is that you are dealing with ["minor" missions,](https://github.com/endless-sky/endless-sky/wiki/CreatingMissions#:~:text=The%20mission%20name%20must,should%20come%20last%20alphabetically.) which don't offer at the same time as non-minor missions, for the sake of the beginning of certain missions series.

Thank you for your kind words and sticking with us. Many more years to come!

**Reply by xX-Dillinger-Xx** [reply]
Not sure if you saw the "timer" part in the OP's question. I thought that some stories had a wait timer. I remember being asked by someone to give them some time to complete their tasks, before returning to a planet. (no spoilers) Actually, I believe there are several.

**Reply by Saugia** [reply]
I had interpreted that as spending time away in space during flight. If OP means mission-wise including landing instances, then yes, there are cases where the missions give you a break, typically a few weeks or months with rare cases being portions of a year or so (all in in-game time). Typically though, those missions mention that there will be a break or time for you to go about yourself.

**Reply by cryptoatropine** [reply]
yes i recall, seeing in free worlds quests, this happening, and in coalition space.. ... after some of battles and the like .. this is not what i meant though, this is 'good' to have... it adds a touch of realism ... what i mean.. is when your done and looking for quests.. 

However if it's just a rnd toss, then it makes sense how it shows up, the rnd on computer is not as rnd as you think ... it always seems to group ....  perhaps there is a way to test is something is active ?? or perhaps block the space port button when something is active?  it looks as though this could be done with the current code,...

A bit off topic here but related to start game play ---- i mean right when you get your ship but after the 'tutorial'

When i start over which i do a lot, to see if i missed things,    i start by helping the guy do the loop, then i get a ramscoop and head to hai space, and build rep there, and save for a heavy hai warship then go back with it for the 'human quests'.
